account-compression/programs/wrapper/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "wrapper"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+crate-type = ["cdylib", "lib"]
+name = "wrapper"
```

Comment:
same as with the other, may as well get rid of this

Reply:
Removed name